src/edit_python_pe/markdown_builder.py

```
@@ -0,0 +1,280 @@
+        return
+
+    # Extract YAML frontmatter
+    yaml_match = re.search(r"---\n(.*?)---\n", content, re.DOTALL)
+    yaml_data = {}
+
+    if yaml_match:
+        try:
+            import yaml
+
+            yaml_data = yaml.safe_load(yaml_match.group(1))
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)